doc/rules/alias/no_alias_functions.rst

```diff
@@ -116,18 +116,14 @@ Rule sets
 
 The rule is part of the following rule sets:
 
-@PHP74Migration:risky
-  Using the `@PHP74Migration:risky <./../../ruleSets/PHP74MigrationRisky.rst>`_ rule set will enable the ``no_alias_functions`` rule with the default config.
```

Comment:
What is the reasoning behind dropping the part "with the default config"? Wasn't it more explicit with it?

Reply:
Yes, it was more explicit, but IMHO unnecessary. I think it's pretty obvious that if you add some rule to the config, without providing options, it runs with the defaults. You can customise it as opt-in, that's why only rules with config override have such info. Rulesets with many rules look redundant with such information displayed again and again.

Reply:
In the example you showed 1st and 4th sets had it with default config which was not so obvious to me.

Can we update it so the sets with default config go first and the others with non-default config?

Reply:
We can do everything but I'm not in favor for that 😅. Rulesets are sorted alphabetically, so  if we split them based on custom config, it will be worse in terms of discoverability (IMHO). To be honest that was my first thought (to split it) but I've changed my mind while implementing 😝.